src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityFinder.java

```diff
@@ -63,82 +65,122 @@ public void configure(EntityFilter filter, boolean showVersions, SelectedHandler
 		this.filter = filter;
 		this.showVersions = showVersions;
 		this.selectedHandler = handler;
+		this.selectedEntity = new ArrayList<Reference>();
+	}
+	
+	public void configureMulti(boolean showVersions, SelectedHandler<List<Reference>> handler) {
+		configureMulti(EntityFilter.ALL, showVersions, handler);
+	}
+	
+
+	public void configureMulti(EntityFilter filter, boolean showVersions, SelectedHandler<List<Reference>> handler) {
+		this.filter = filter;
+		this.showVersions = showVersions;
+		this.selectedMultiHandler = handler;
+		this.selectedEntity = new ArrayList<Reference>();
 	}
 	
 	@Override
 	public void setSelectedEntity(Reference selected) {
 		synAlert.clear();
-		selectedEntity = selected;
+		selectedEntity.clear();
+		selectedEntity.add(selected);
+	}
+	
+	@Override
+	public void setSelectedEntity(List<Reference> selected) {
+		synAlert.clear();
+		selectedEntity.clear();
+		selectedEntity.addAll(selected);			
+		
 	}
 	
 	@Override
 	public void okClicked() {
 		synAlert.clear();
 		//check for valid selection
-		if (selectedEntity == null || selectedEntity.getTargetId() == null) {
+		if (selectedEntity == null || selectedEntity.get(0).getTargetId() == null) {
```

Comment:
When nothing has been selected, selectedEntity contains one Reference with a null target Id?  I would have thought selectedEntity (which I think should be renamed to selectedEntities) would be empty.